GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -251,11 +279,16 @@ public typealias CameraScreenFailureBlock = (_ error: GiniVisionError) -> Void
         NotificationCenter.default.removeObserver(self)
     }
     
-    fileprivate func didPick(validatedDocuments documents: [GiniVisionDocument]) {
+    fileprivate func didPick(validatedDocuments documents: [GiniVisionDocument],
+                             completion: DocumentPickerCompletion?) {
         if let delegate = delegate {
-            delegate.camera(self, didCaptureDocuments: documents)
+            delegate.camera(self, didCaptureDocuments: documents, completion: completion)
         } else if let firstDocument = documents.first {
+            completion?(nil, nil)
             successBlock?(firstDocument)
+            if documents.count > 1 {
+                // TODO: Show a warning in the logger
+            }
         } else {
             print("It has not been specified a CameraViewControllerDelegate")
```

Comment:
a `CameraViewControllerDelegate ` has not been specified.
This sounds waaay better. In fact, yours is probably not even correct English.